pkg/api/validation/validation.go

```diff
@@ -30,12 +30,29 @@ import (
 	"github.com/golang/glog"
 )
 
+const MinPort int = 1
```

Comment:
I think all of these symbols can be internal, can't they?  Lower-case leading character..